l10n-pocket/en/pocket/tos.ftl

```diff
@@ -0,0 +1,128 @@
+
+pocket-tos-the-videos-articles = The videos, articles and other content made available on third-party websites may be protected by copyright and other applicable laws. Some content may not be reproduced, used to prepare derivative works, distributed, performed publicly or displayed publicly without the written permission of the copyright holder, except to the extent allowed under the "fair use" provisions of the U.S. copyright laws, other limitations on exclusive copyrights in the U.S. copyright laws, or comparable provisions of foreign laws. Other content may have re-use permissions attached. Please refer to the website you were visiting when you saved such content for information that may be applicable to such content.
+pocket-tos-you-represent = You represent and agree that the { -brand-name-pocket } application will be used solely for your personal-noncommercial use with respect to content of websites that you have the lawful right to access, and that the { -brand-name-pocket }  application will not be used in violation of the restrictions posted on the website where such content was published. Some websites include security measures designed to prevent access, and you agree not to take any steps to defeat any such security measures. We do not necessarily monitor any materials posted, transmitted, or saved to or with the { -brand-name-pocket-technologies } . By posting, sharing or saving any videos, articles or content, you represent that doing so does not infringe any third party’s copyrights, trademarks, privacy rights or other intellectual property or legal rights of any kind. If notified that any user has posted, shared or saved any information or materials which allegedly do not conform to this Agreement, we may in our sole discretion investigate the allegation and determine whether to take any other actions, including, but not limited to, whether to remove or request the removal of the information or materials posted, shared or saved by a user. We also reserve the right to terminate the account of any user who transfers or saves content in violation of this Agreement. We shall have no liability or responsibility to users for performance or nonperformance of such activities. You may be subject to civil and criminal penalties, including without limitation monetary damages, if you violate the terms of this Agreement or infringe any third party’s legal rights.
```

Comment:
**Nit:** remove the space before the period before `By posting, sharing or saving any videos`:
```suggestion
pocket-tos-you-represent = You represent and agree that the { -brand-name-pocket } application will be used solely for your personal-noncommercial use with respect to content of websites that you have the lawful right to access, and that the { -brand-name-pocket }  application will not be used in violation of the restrictions posted on the website where such content was published. Some websites include security measures designed to prevent access, and you agree not to take any steps to defeat any such security measures. We do not necessarily monitor any materials posted, transmitted, or saved to or with the { -brand-name-pocket-technologies }. By posting, sharing or saving any videos, articles or content, you represent that doing so does not infringe any third party’s copyrights, trademarks, privacy rights or other intellectual property or legal rights of any kind. If notified that any user has posted, shared or saved any information or materials which allegedly do not conform to this Agreement, we may in our sole discretion investigate the allegation and determine whether to take any other actions, including, but not limited to, whether to remove or request the removal of the information or materials posted, shared or saved by a user. We also reserve the right to terminate the account of any user who transfers or saves content in violation of this Agreement. We shall have no liability or responsibility to users for performance or nonperformance of such activities. You may be subject to civil and criminal penalties, including without limitation monetary damages, if you violate the terms of this Agreement or infringe any third party’s legal rights.
```